### ISSUE TITLE
구법과 현행법령의 명칭이 다를때

### ISSUE THREAD

**Issue by tenystory** [no05s/pdf-support#12]
예를 들어 구 파산법 9조라고 하면
파산법 9조로 작업해야 하는지
현행법령 명칭인 채무자 회생 및 파산에 관한 법률 9조로 작업해야 하는지 알고 싶어요.

![image](https://user-images.githubusercontent.com/51429942/59428713-fc042480-8e18-11e9-9c11-22a81f300c1e.png)
 구 대한민국헌법 (헌법 제9호로 개정되기 전의 것 유신헌법이라 한다) 제53조의 근거한
에서 유신헌법은 법이 아니라 작업 안하는걸로 알고 있는데 구 빼고 대한민국 헌법 53조로 작업해야 하는건가요?

**Comment by dhkim11** [comment]
네, 법령명 앞에 언급된 '구 *법' 에서 '구'는 무시하고 현행 법령명(풀네임)으로 적용해서 입력하면 됩니다.
법령명에서 파산법은 줄임말 법령명입니다. 입력시에 줄임말이 아닌 풀네임 법령명으로 입력해야 합니다.
구 파산법 9조 -> 채무자 회생 및 파산에 관한 법률 9조
구 대한민국헌법 (헌법 제9호로 개정되기 전의 것 유신헌법이라 한다) 제53조 -> 대한민국헌법 53조

**Comment by tenystory** [comment]
그런데요. 파산법9조와 현행 채무자회생법 9조와 내용이 다른 경우는요? 유신헌법 같은 경우도 현재 대한민국 헌법 53조와 그 내용이 다른것 같아요. 이런 경우로 그냥 동일한 기준으로 엑셀 작업 하나요?

**Comment by dhkim11** [comment]
네, 현행 법령 기준으로 작업하면 됩니다. 
단, 원문에 명시된 법령명, 조, 항, 호, 목 정보는 그대로이며 법령명은 현행 법령명 그대로 입력하시면 됩니다.
